src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -231,6 +231,22 @@ This feature will be automatically enabled for you once your GitHub integration
 
 ![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)
 
+### Missing Member Detection
+
+If your organization has users committing to Github repositories linked to Sentry that are not members of your organization, Sentry will remind organization owners and managers to invite these users through a banner on the members list page.
+
+![Github missing members invite banner](github-missing-members-banner.png)
+
+The banner will show the 5 most active contributors. To see all of the missing members (up to 50), click **View All** on the banner. To snooze the banner, click the ellipsis > **Hide Missing Members**.
+
+// TODO: modal image
+
+You can select multiple users and assign roles and teams before inviting them through the modal.
```

Comment:
Maybe this would be more obvious in practice, but it's unclear to me how you'd select multiple users and what the flow would be like from the image. Do you need to click "Invite Members" to do this?

Reply:
there are checkboxes on the modal (see https://github.com/getsentry/sentry/pull/55003) and then there's an "Invite Members" at the bottom